Gemfile.lock

```diff
@@ -1,256 +0,0 @@
-GIT
```

Comment:
`Gemfile.lock` is important for tracking your dependencies. Any reason this was deleted?